packages/rum-core/src/performance-monitoring/breakdown.js

```diff
@@ -0,0 +1,130 @@
+/**
+ * MIT License
+ *
+ * Copyright (c) 2017-present, Elasticsearch BV
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ *
+ */
+
+import { getDuration } from '../common/utils'
+import { PAGE_LOAD } from '../common/constants'
+
+/**
+ * Page load transaction breakdown timings
+ *
+ * Interested events from the Navigation timing API
+ */
+const pageLoadBreakdowns = [
+  ['fetchStart', 'requestStart', 'Network'],
+  ['requestStart', 'responseStart', 'TTFB'],
```

Comment:
I was choosing between TTFB/Server response time. Probably later is better since TTFB might confuse users. I was actually using the chrome devtools timing viewer which more users are familiar with. 
<img width="1144" alt="Screenshot 2019-09-13 at 10 19 32" src="https://user-images.githubusercontent.com/3902525/64847749-ff719c80-d60f-11e9-98e8-b79ce80999a3.png">

Reply:
Yeah, I agonised over this one a bit. I think in devtools it's clearer since it's a timeline, whereas in our breakdown it'll be freestanding.